mlir/test/CMakeLists.txt

```diff
@@ -28,6 +27,16 @@ set(MLIR_ROCM_WRAPPER_LIBRARY_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
 set(MLIR_SPIRV_WRAPPER_LIBRARY_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
 set(MLIR_VULKAN_WRAPPER_LIBRARY_DIR ${CMAKE_LIBRARY_OUTPUT_DIRECTORY})
 
+# Passed to lit.site.cfg.py.in to set the -x2 flag for the mlir E2E tests.
+if(MLIR_MIOPEN_DRIVER_XDLOPS_TEST_ENABLED)
+  set(MLIR_XDLOPS "-x2")
+endif()
```

Comment:
For safety, shouldn't there be an else branch that sets `MLIR_XDLOPS` to either the empty string  or an xdlops-disabling flag when the xdlops tests aren't enabled?